File: app/src/main/java/com/hypnoweb/hypnowebapp/uibinders/SplashScreenBindingAdapter.java
package com.hypnoweb.hypnowebapp.uibinders;

import android.databinding.BindingAdapter;
import android.support.v4.widget.ContentLoadingProgressBar;

/**
 * Created by stefan on 12/3/2017.
 */

public class SplashScreenBindingAdapter {


    @BindingAdapter("loading")
    public static void setLoadingContentLoadingProgressBar(ContentLoadingProgressBar contentLoadingProgressBar, Boolean loading) {
        if (contentLoadingProgressBar != null) {
            contentLoadingProgressBar.show();
            contentLoadingProgressBar.setIndeterminate(true);
        }

    }
}

File: app/src/main/java/com/hypnoweb/hypnowebapp/uibinders/SplashScreenBindingAdapter.kt
package com.hypnoweb.hypnowebapp.uibinders

import android.databinding.BindingAdapter
import android.support.v4.widget.ContentLoadingProgressBar


@BindingAdapter("loading")
fun setLoadingContentLoadingProgressBar(contentLoadingProgressBar: ContentLoadingProgressBar?, loading: Boolean?) {
    if (contentLoadingProgressBar != null) {
        contentLoadingProgressBar.show()
        contentLoadingProgressBar.isIndeterminate = true
    }
}

@BindingAdapter("loading2")
fun setLoadingContentLoadingProgressBar2(contentLoadingProgressBar: ContentLoadingProgressBar?, loading: Boolean?) {
    if (contentLoadingProgressBar != null) {
        contentLoadingProgressBar.show()
        contentLoadingProgressBar.isIndeterminate = true
    }
}
